packages/cli/src/lib/pg/host.ts

```diff
@@ -1,4 +1,4 @@
 export default function () {
-  const host = process.env.HEROKU_DATA_HOST || process.env.HEROKU_POSTGRESQL_HOST
+  const host = process.env.HEROKU_DATA_HOST
   return host ? host : 'api.data.heroku.com'
```

Comment:
perhaps this should align with the pattern in `packages/cli/src/lib/redis/api.ts`
```suggestion
  const host = process.env.HEROKU_DATA_HOST || 'api.data.heroku.com'
```